cmd/internal/find/primary/meta/arrayPredicate.go

```diff
@@ -0,0 +1,145 @@
+package meta
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/puppetlabs/wash/cmd/internal/find/primary/errz"
+)
+
+// ArrayPredicate => EmptyPredicate      |
+//                   ‘[' ‘]’ Predicate   |
```

Comment:
Thinking we could express `some` as `[?]` to make it more explicit.

Reply:
Hmm. Not no. This is a bit distinct from jq in that it's explicitly a filter, rather than a map operation.

Reply:
Yeah, I think `[?]` is better. I'll update the PR later with that modification.